pkg/api/v1beta3/types.go

```diff
@@ -1609,3 +1610,19 @@ type SecretList struct {
 
 	Items []Secret `json:"items" description:"items is a list of secret objects"`
 }
+
+// ServerStatus (and ServerStatusList) holds the cluster validation info.
+type ServerStatus struct {
+	Component  string       `json:"component,omitempty"`
+	Health     string       `json:"health,omitempty"`
+	HealthCode probe.Result `json:"healthCode,omitempty"`
+	Msg        string       `json:"msg,omitempty"`
+	Err        string       `json:"err,omitempty"`
```

Comment:
Did you consider embedding Status instead? There should at least be a compelling reason why this is significantly different than Status.